stdlib/public/Concurrency/Task.swift

```diff
@@ -143,57 +159,58 @@ extension Task {
 
 @available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *)
 extension Task {
-  /// A task handle refers to an in-flight `Task`,
-  /// allowing for potentially awaiting for its result or Cancelling it.
+  /// An affordance to interact with an active task.
+  ///
+  /// You can use a task's handle to wait for its result or cancel the task.
   ///
-  /// It is not a programming error to drop a handle without awaiting or cancelling it,
-  /// i.e. the task will run regardless of the handle still being present or not.
-  /// Dropping a handle however means losing the ability to await on the task's result
-  /// and losing the ability to cancel it.
+  /// It's not a programming error to discard a task's handle without awaiting or canceling the task.
+  /// A task runs whether or not you still have its handle stored somewhere.
+  /// However, if you discard a task's handle, you give up the ability
+  /// to wait for that task's result or cancel the task.
   public struct Handle<Success, Failure: Error>: Sendable {
     internal let _task: Builtin.NativeObject
 
     internal init(_ task: Builtin.NativeObject) {
       self._task = task
     }
 
-    /// Returns the `Task` that this handle refers to.
+    /// The task that this handle refers to.
     public var task: Task {
       Task(_task)
     }
 
-    /// Wait for the task to complete, returning (or throwing) its result.
+    /// Wait for the task to complete, returning its result or throw an error.
     ///
-    /// ### Priority
-    /// If the task has not completed yet, its priority will be elevated to the
+    /// If the task hasn't completed yet, its priority is elevated to the
     /// priority of the current task. Note that this may not be as effective as
-    /// creating the task with the "right" priority to in the first place.
+    /// creating the task with the right priority to in the first place.
     ///
-    /// ### Cancellation
-    /// If the awaited on task gets cancelled externally the `get()` will throw
-    /// a cancellation error.
+    /// If task is canceled, this method throws a cancellation error.
```

Comment:
we lost the "externally" here:

```suggestion
    /// If task is canceled externally, this method throws a cancellation error.
```

Although I'll double check if we actually do so... I don't think we do to be honest 🤔

Reply:
Are we sure we want those semantics...? I don't think we actually do this today; it'd require "check if the task we refer to was cancelled and throw magically here" I'm not sure this is actually easy to know vs. the "internally"... 😖 

@DougGregor @jckarter